tests/integration/integration_test.go

```diff
@@ -291,3 +270,35 @@ func Test_EventFilters(t *testing.T) {
 		})
 	}
 }
+
+const (
+	doMagicWrite = "do_magic_write"
+	doLs         = "do_ls"
+	doDockerRun  = "do_docker_run"
+)
+
+// forkAndExecFunction runs a function in `tester.sh` in it's own system process.
+// This is so Tracee running in the current pid can pick the command up.
+// It returns the output of the process and a possible error.
+func forkAndExecFunction(funcName string) ([]byte, error) {
```

Comment:
Small suggestion, we can do
`type testFunc string`
and then define the consts above as such and `funcName` as such, which would prevent passing something arbitrary here in the future by mistake.

Reply:
Sounds good!